app/models/comment.rb

```diff
@@ -1,4 +1,17 @@
 class Comment < ApplicationRecord
   belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_create :increment_post_comments_counter
```

Comment:
- Please use the after_save method to call the increment_function, as we only want to increment the attribute only when the data is saved in the database, please fix this 👍
- Please update your `like` model with a function that increments the likes_counter  attribute in the `Post` model, please do not forget to call the function after the data has been saved to the database 🙏